src/sentry/static/sentry/app/components/events/interfaces/togglableAddress.tsx

```diff
@@ -108,6 +110,7 @@ const AddressText = styled('span')<Partial<Props>>`
       return 'none';
     }
   }};
+  padding-left: ${p => (p.canBeConverted ? null : '18px')};
```

Comment:
Just curious, but where do those 18px come from? Shouldn't this be some constant?

Reply:
Yeah, I was thinking about the same thing, but It's a width of a toggle icon that is not there in this case (since we can't convert between two formats) - so probably nothing that can be reused...